### DISCUSSION TITLE
Documentation Error 404

### DISCUSSION THREAD

**Discussion by alexcbb** [haosulab/ManiSkill#289] [General]
Hello, I've tried to access to the documentation of ManiSkill and it seems that I directly get an error 404. Is there an issue with the website ?

**Comment by StoneT2000** [comment]
We are in the middle of updating to ManiSkill 3 and have moved documentation locations.

If you are looking for ManiSkill 2 documentation you can find it here: https://maniskill.readthedocs.io/en/v0.5.3-rtd/

Otherwise I recommend waiting a few days, ManiSkill 3 has a ton of upgrades (and breaking changes) to enable a number performance improvements and features.
